docs/styles.css

```diff
@@ -0,0 +1,216 @@
+* {
+    box-sizing: border-box;
+    padding:0;
+    margin:0;
+    
+    
+    
+}
+
+body{
+    height: 100vh;
+    background-image: url(assets/bg-desktop.png);
+    background-size: 100vw 50vh ;
+    background-repeat: no-repeat;
+    background-position: bottom;
+    background-color:hsl(229, 57%, 11%); 
+    display:flex;
```

Comment:
turning the body to be display flex can be very buggy when you want adding more and more items that not need to be in the scope of your flexbox container